test/socket.router.js

```diff
@@ -42,15 +44,15 @@ describe('socket.router', function(){
 
```

Comment:
Not related to this PR, but I'll use it to ask a question...
Why do we need to test "// should throw an error on unroutable msgs if mandatory = 1 and no error handler is set" ?
Testing that we emit an "error" message should be enough, the "convert" from unhandled "error" event to exception is node Emitter behavior, and we have no intention on testing node Emitter here... no?